src/components/Footer/Footer.jsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import List from "@mui/material/List";
+import FooterItem from "./FooterItem";
+
+function Footer() {
+  return (
+    <List>
```

Comment:
Please change https://mui.com/material-ui/react-menu/ -  <MenuList> instead of List components

Inside of MenuList you should take List of Items, and render <MenuItem>

Reply:
In Jira was a link mui.com/material-ui/react-list/, so I assumed I have to use List.

Changed to MenuList and MenuItem, it works but I see errors and warnings in a console:
"Warning: Expected server HTML to contain a matching <ul> in <html>", "Uncaught Error: Hydration", "Warning: An error occurred during hydration. The server HTML was replaced with client content in <#document>", "Warning: validateDOMNesting(...): <ul> cannot appear as a child of <html>", "Uncaught Error: Hydration failed because the initial UI does not match what was rendered on the server", "Uncaught Error: There was an error while hydrating. Because the error happened outside of a Suspense boundary, the entire root will switch to client rendering".

Reply:
Yes, sorry, my bad, mui.com/material-ui/react-list/ should be MenuList component.

Error in console, is because I suppose u are rendering one tag in another not supported tag

You can get more info about it from Next.js docs - https://nextjs.org/docs/messages/react-hydration-error